mongo/integration/unified/collection_operation_execution.go

```diff
@@ -454,7 +475,14 @@ func executeEstimatedDocumentCount(ctx context.Context, operation *operation) (*
 	}
 
 	opts := options.EstimatedDocumentCount()
-	elems, _ := operation.Arguments.Elements()
+	var elems []bson.RawElement
+	// Some estimatedDocumentCount operations in the unified test format have no arguments.
```

Comment:
Most of the `estimatedDocumentCount` operations in [`unified/estimatedDocumentCount`](https://github.com/mongodb/mongo-go-driver/blob/master/data/crud/unified/estimatedDocumentCount.yml) do not have an `arguments` field, so I've added a special conditional here. I could add this to all `execute*` functions, but it should be easy to add later on if we encounter other operations with no arguments.

I checked to see who wrote these tests and decided to recklessly not include the `arguments` field, and it was me a year ago 😢 .